docs/sdks/js-reference-guide/js-sdk-authenticateOtp.md

```diff
@@ -0,0 +1,64 @@
+---
+title: authenticateOtp
+id: js-reference-authenticateOtp
+description: ''
+slug: /js-reference-authenticateOtp
+keywords:
+ - javascript sdk
+pagination_next: null
+pagination_prev: null
+last_update:
+   date: 07/25/2023
+   author: William May
+draft: false
+doc_type: reference
+displayed_sidebar: sdkSidebar
+---
+
+The **authenticateOtp** function enables an app using the Beyond Identity Javascript SDK to initiate authentication using an OTP. The OTP will be sent to the provided email address.
```

Comment:
Might be helpful to mention that this is used in conjunction with redeemOtp

Reply:
Added.